build.gradle

```diff
@@ -96,6 +96,7 @@ allprojects {
             palantirJavaFormat('2.26.0')
             trimTrailingWhitespace()
             endWithNewline()
+            ratchetFrom('origin/main')
```

Comment:
What if my main remote is not named origin? I personally hare "Wynntils" as origin..

Reply:
I dunno... :-& 

This seems to be tricky to get right. I'd like to see spotless fix years in the header, but I don't know how to get there.